config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :exams
```

Comment:
Could you please put this with the rest of the resources? Ideally alphabetically. :P